schema.sql

```diff
@@ -1,13 +1,32 @@
 /* Database schema to keep the structure of entire database. */
-CREATE DATABASE vet_clinic;
-
-CREATE TABLE animals(
-  id INT PRIMARY KEY NOT NULL,
-  name TEXT NOT NULL,
-  date_of_birth DATE,
-  escape_attempts INT,
-  neutered BOOLEAN,
-  weight_kg DECIMAL
+
```

Comment:
**Changes Requested**
- [ ] You should create tables `vet, visit` and `specialization` according to the project requirement. I am afraid I wasn't able to find any. Let us please make these changes as they're the core project requirements.